contrib/features/TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+# Change Request
```

Comment:
lets get a purpose statement in this, as well as a short description on how to use it and the process.